gambatte_libretro.info

```diff
@@ -1,5 +1,5 @@
 # Software Information
-display_name = "Nintendo - Game Boy / Color (Gambatte)"
+display_name = "Nintendo - GB/GBC (Gambatte)"
```

Comment:
I still believe acronyms are a bad idea here

Reply:
how would you feel about "Game Boy/GBC"? I feel like GBC and GBA are distinct and common enough (and also long enough) to be recognizable on their own.